vulture_os/system/cluster/models.py

```diff
@@ -490,7 +489,6 @@ def get_backends_enabled(self):
         """ Return all tuples (family, proto, ip, port) for each enabled Backends on this node """
         from applications.backend.models import Backend
         # !!! REQUIRED BY listener_set !
```

Comment:
Same as bellow.

Reply:
import moved back to the beginning of the file